doc/userguide/rules/datasets.rst

```diff
@@ -162,12 +162,56 @@ Syntax::
 
     dataset-remove <set name> <set type> <data>
 
+set name
+  Name of an already defined dataset
+type
+  Data type: string, md5, sha256, ipv4, ip
+data
+  Data to remove in serialized form (base64 for string, hex notation for md5/sha256, string representation for ipv4/ipv6)
+
+dataset-clear
+~~~~~~~~~~~~~
+
+Unix Socket command to remove all data from a set. On success, the removal becomes
+active instantly.
+
+Syntax::
+
+    dataset-clear <set name> <set type>
+
+set name
+  Name of an already defined dataset
+type
+  Data type: string, md5, sha256
+
+dataset-lookup
+~~~~~~~~~~~~~~
+
+Unix Socket command to test if data is in a set. 
+
+Syntax::
+
+    dataset-lookup <set name> <set type> <data>
+
 set name
   Name of an already defined dataset
 type
   Data type: string, md5, sha256
 data
-  Data to remove in serialized form (base64 for string, hex notation for md5/sha256)
+  Data to test in serialized form (base64 for string, hex notation for md5/sha256)
```

Comment:
Are we missing info about ipv4 and ip here? (asking because they have been added on the previous sections)

Reply:
Good catch